Library/Homebrew/cmd/commands.rb

```diff
@@ -69,5 +69,7 @@ def find_internal_commands(directory)
         cmds << f.basename.to_s.sub(/\.(?:rb|sh)$/, "") if f.file?
       end
     end
+
+    private :internal_commands, :internal_developer_commands, :external_commands, :find_internal_commands
```

Comment:
A common way to define `private` methods is to put those below the keyword, e.g.:

```ruby
class MyClass

  def my_public_method
  end
  
  def my_other_public_method
  end

  private

  def this_one_is_private
  end

  def this_one_too
  end
end
```